### PR TITLE
Pull adjustments out of Cypress 11 branch

### PR DESCRIPTION
**Pull request by seriouslysean**
## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle#linting-formatting))
- For New Features:
  - [x] Have tests been added to cover any new features or fixes?
  - [x] Has the documentation been updated accordingly?

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -73,6 +73,13 @@ module.exports = {
         'no-undef': 'warn',
         'no-prototype-builtins': 'warn',
       },
+      globals: {
+        Card: true,
+        Season: true,
+        Match: true,
+        User: true,
+        Game: true,
+      },
```
Comment (seriouslysean) [inline .eslintrc.js:82]: Missing global allowances.

File: api/policies/isLoggedIn.js [hunk 2]
```diff
@@ -3,20 +3,24 @@
...
   }
 
   // User is not allowed
   // (default res.forbidden() behavior can be overridden in `config/403.js`)
-  return res.forbidden({ message: 'You must be logged in to perform this action.' });
+  return res.forbidden({
+    message: 'You must be logged in to perform this action.',
+    path,
+  });
 };
```
Comment (seriouslysean) [inline api/policies/isLoggedIn.js:26]: Added some nice to have changes to the logged in policy middleware.

File: client/js/plugins/sails.js [hunk 3]
```diff
@@ -7,13 +7,33 @@ const { isProd } = require('../../../utils/config-utils');
...
+
+      // If no connection after threshold, reject the promise to prevent an infinite loop
+      if (tries >= MAX_TRIES) {
+        clearInterval(interval);
+        reject();
+      }
+
+      tries += 1;
+    }, INTERVAL);
+  });
```
Comment (seriouslysean) [inline client/js/plugins/sails.js:30]: Rewrote the reconnect logic so we can properly await it when we need to use it in our code (with proper timeout support).

File: client/js/store/modules/auth.js [hunk 4]
```diff
@@ -21,9 +21,9 @@ async function handleLogin(context, username, password, signup = false) {
     if (response.status !== 200) {
       throw data.message;
     }
+    await reconnectSockets();
```
Comment (seriouslysean) [inline client/js/store/modules/auth.js:24]: Moved this to before we commit a success for better sequencing.

File: tests/e2e/support/index.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
+import 'cypress-fail-fast';
```
Comment (seriouslysean) [inline tests/e2e/support/index.js:1]: This is to better help Cypress not eat our GitHub action minutes.

File: vue.config.js [hunk 6]
```diff
@@ -50,6 +50,10 @@ module.exports = {
         target: 'http://localhost:1337',
         changeOrigin: true,
       },
+      // Required for the health response to work on the client
+      '/health': {
+        target: 'http://localhost:1337',
+      },
```
Comment (seriouslysean) [inline vue.config.js:56]: Fixes the local health path so it actually works.

Comment (itsalaidbacklife) [reply vue.config.js:56]: Resolves #181

**Review by itsalaidbacklife — APPROVED** [review]
Looks good; no changes needed

**Pull request merged**